### ISSUE TITLE
2 errors, while compiling

### ISSUE THREAD

**Issue by KrYpToDeN** [samp-incognito/samp-streamer-plugin#143]
Hello.
I got two errors, when i tried to compile your plugin on Linux.
See picture (white box).
I hope, you will check it.

![default](https://cloud.githubusercontent.com/assets/10742701/18576570/170496aa-7c26-11e6-9d6a-e820130747ea.png)

**Comment by oscar-broman** [comment]
I got the same problems. Also I'm running into problems with openssl while compiling premake5 (it's not supported on Arch yet) so that doesn't make things easier.

**Comment by samp-incognito** [comment]
Thanks.

@oscar-broman: You could just try the makefile in the repository here.

**Comment by IstuntmanI** [comment]
Well, there are few warnings too:
![screenshot_1](https://cloud.githubusercontent.com/assets/6208948/18765417/4b548a9e-811e-11e6-977b-1d1efb94ff41.png)

![screenshot_2](https://cloud.githubusercontent.com/assets/6208948/18765434/66e82590-811e-11e6-9891-f1b30b231691.png)

**Comment by samp-incognito** [comment]
That's a pretty valid point in the last image, as those casts shouldn't be there. It doesn't really affect anything, though.

The sampgdk.h/amx.h warnings only happen on older compiler versions, I think.

**Comment by IstuntmanI** [comment]
I checked and it looked like I was indeed using GNU Make 3.81 (released in ~2006) (I saw somewhere that Ubuntu has a problem and it is using 3.81, I'm using Bash on Ubuntu on Windows 10), so I tried to update it to the latest. Finally, I got it to update to 4.2. But those warnings are still appearing.

By the way, I compiled the latest commit both on Linux (only those amx.h warnings) and Windows (no warnings). In case someone wants them, here they are:
https://mega.nz/#!7sY2Waxa!cFiAOrRqxRAuDfN3mLlGkZDoF2S0efPsFOSSXcgifB4
